android/src/main/java/io/sentry/RNSentryModule.java

```diff
@@ -146,6 +146,11 @@ public void captureEnvelope(String envelope, Promise promise) {
         promise.resolve(true);
     }
 
+    @ReactMethod
+    public void getStringBytesLength(String str) {
```

Comment:
I'd assume that you need a `Promise promise` here

Reply:
Didn't add since there's nothing async here, but no idea if it's needed.
there's `setLogLevel` which doesn't take a Promise. Tomorrow when Vienna is awake we can discuss this change properly. Wanted to make a pointer to where the problem is and suggest some fix

Reply:
yep, got it, but if you don't have a `return` type on the method declaration, it won't work, got your point, will discuss with @HazAT today, thanks for checking it out

Reply:
yeah next time I'll pay attention what I'm doing :)